pkg/services/container/morph/executor.go

```diff
@@ -274,3 +291,41 @@ func (s *morphExecutor) GetExtendedACL(_ context.Context, body *container.GetExt
 
 	return res, nil
 }
+
+func (s *morphExecutor) validateToken(t *sessionV2.Token, cID *refs.ContainerID, op sessionV2.ContainerSessionVerb) error {
```

Comment:
worth unit testing

Reply:
not valuable to me cause i had to fix tests a little (the only one in that pkg named "InvalidToken") to make them feel good. also, that is V2 level so most of the tests look like "create byte array one, create byte array two, make sure byte array one does not equal the second one". but ok, added